### DISCUSSION TITLE
bad firmware?

### DISCUSSION THREAD

**Discussion by Stanojoski** [roleoroleo/sonoff-hack#170] [General]
I am unable to configure the camera When powered, it does not give me any voice instructions. I see the Wi-Fi HAP-EWLK..., but when I try to connect it to ewelink, it gives me Error 6750 Pairing failed. I suspect that the firmware is bad. When inserting this roleoroleo sonoff hack, the camera turns right and up. And after that, everything is the same.  Any ideas or any firmware I can try?

**Comment by roleoroleo** [comment]
If you dump the fw, I will compare it with other firmwares:
https://github.com/roleoroleo/sonoff-hack/wiki/Dump-your-backup-firmware-(SD-card)

**Reply by Stanojoski** [reply]
Thanks for answering. I ran the instructions, and here are the files. 

LINK REMOVED

SHA256:    0076EB5D8E37E94D55B5138026A96CFB177A28D8EE535FF6FECF7230573EDD99

**Reply by roleoroleo** [reply]
Your fw version is new for me. I never tested it with my hack.
But this is not related to your problem. The cam should work without the need of the hack.
Is a new fw version (Sonoff I mean) available? Maybe a bug...

**Reply by Stanojoski** [reply]
The hack was never a problem. The camera was not working from before.
I cannot connect it at all to the network. When plugged in it does nothing, no voice as the other cameras. With Ewelink I try to add it to the network, but always receive the  Error 6750 Pairing failed.

I hoped to find some firmware that is working to upload it and test it.

**Reply by roleoroleo** [reply]
Sorry but I don't have a version to give you for your model.

**Reply by roleoroleo** [reply]
Maybe here...
https://github.com/roleoroleo/sonoff-hack/issues/85#issuecomment-1217990147
